system_tests/eth_config_test.go

```diff
@@ -0,0 +1,112 @@
+				"ArbInfo":               common.HexToAddress("0x0000000000000000000000000000000000000065"),
+				"ArbNativeTokenManager": common.HexToAddress("0x0000000000000000000000000000000000000073"),
+				"ArbOwner":              common.HexToAddress("0x0000000000000000000000000000000000000070"),
+				"ArbOwnerPublic":        common.HexToAddress("0x000000000000000000000000000000000000006b"),
+				"ArbRetryableTx":        common.HexToAddress("0x000000000000000000000000000000000000006e"),
+				"ArbStatistics":         common.HexToAddress("0x000000000000000000000000000000000000006f"),
+				"ArbSys":                common.HexToAddress("0x0000000000000000000000000000000000000064"),
+				"ArbWasm":               common.HexToAddress("0x0000000000000000000000000000000000000071"),
+				"ArbWasmCache":          common.HexToAddress("0x0000000000000000000000000000000000000072"),
+				"ArbosActs":             common.HexToAddress("0x00000000000000000000000000000000000a4b05"),
+				"ArbosTest":             common.HexToAddress("0x0000000000000000000000000000000000000069"),
+				"BLAKE2F":               common.HexToAddress("0x0000000000000000000000000000000000000009"),
+				"BLS12_G1ADD":           common.HexToAddress("0x000000000000000000000000000000000000000b"),
+				"BLS12_G1MSM":           common.HexToAddress("0x000000000000000000000000000000000000000c"),
+				"BLS12_G2ADD":           common.HexToAddress("0x000000000000000000000000000000000000000d"),
+				"BLS12_G2MSM":           common.HexToAddress("0x000000000000000000000000000000000000000e"),
+				"BLS12_MAP_FP2_TO_G2":   common.HexToAddress("0x0000000000000000000000000000000000000011"),
+				"BLS12_MAP_FP_TO_G1":    common.HexToAddress("0x0000000000000000000000000000000000000010"),
+				"BLS12_PAIRING_CHECK":   common.HexToAddress("0x000000000000000000000000000000000000000f"),
+				"BN254_ADD":             common.HexToAddress("0x0000000000000000000000000000000000000006"),
+				"BN254_MUL":             common.HexToAddress("0x0000000000000000000000000000000000000007"),
+				"BN254_PAIRING":         common.HexToAddress("0x0000000000000000000000000000000000000008"),
+				"ECREC":                 common.HexToAddress("0x0000000000000000000000000000000000000001"),
+				"ID":                    common.HexToAddress("0x0000000000000000000000000000000000000004"),
+				"KZG_POINT_EVALUATION":  common.HexToAddress("0x000000000000000000000000000000000000000a"),
+				"MODEXP":                common.HexToAddress("0x0000000000000000000000000000000000000005"),
+				"P256VERIFY":            common.HexToAddress("0x0000000000000000000000000000000000000100"),
+				"RIPEMD160":             common.HexToAddress("0x0000000000000000000000000000000000000003"),
+				"SHA256":                common.HexToAddress("0x0000000000000000000000000000000000000002"),
+			},
+			SystemContracts: map[string]common.Address{
+				"BEACON_ROOTS_ADDRESS":                    common.HexToAddress("0x000f3df6d732807ef1319fb7b8bb8522d0beac02"),
```

Comment:
I wasn't sure if it was important to modify these for Arbitrum chains.
I suppose it must be, since some of them don't make sense.

Thoughts?